hoomd/md/pair/aniso.py

```diff
@@ -328,6 +328,54 @@ class ALJ(AnisotropicPair):
     only ``vertices`` creates a convex polytope (set ``vertices`` and ``faces``
     to empty lists to create the ellipsoid).
 
+    Important:
+        `ALJ` implicitly rounds the given shape by :math:`\sigma_c` via the WCA
+        or Lennard-Jones potential.
+
+    .. rubric:: Choosing `r_cut <hoomd.md.pair.Pair.r_cut>`:
+
+    Set `r_cut <hoomd.md.pair.Pair.r_cut>` for each pair of particle types so
+    that `ALJ` can compute interactions for all possible relative placements
+    and orientations of the particles. The furthest apart two particles can be
+    while still interacting depends on the value of ``alpha`` (where :math:`R_i`
+    is the circumsphere radius of the particle with type :math:`i`:):
+
+    * For alpha=0:
+
+      .. math::
+
+        r_{\mathrm{cut},ij} = \max \left( \frac{2^\frac{1}{6}}{2}
+        (\sigma_i + \sigma_j), R_i + R_j + R_{\mathrm{rounding},i} +
+        R_{\mathrm{rounding},j} + \frac{2^\frac{1}{6}}{2}
+        (\beta_i \cdot \sigma_i + \beta_j \cdot \sigma_j) \right)
+
+    * For alpha=1:
+
+      .. math::
+
+            r_{\mathrm{cut},ij} = \max \left( \frac{2.5}{2}
+            (\sigma_i + \sigma_j),  R_i + R_j  + R_{\mathrm{rounding},i} +
+            R_{\mathrm{rounding},j}+ \frac{2^\frac{1}{6}}{2}
+            (\beta_i \cdot \sigma_i + \beta_j \cdot \sigma_j) \right)
+
+    * For alpha=2:
+
+      .. math::
+
+            r_{\mathrm{cut},ij} = \max \left( \frac{2^\frac{1}{6}}{2}
+            (\sigma_i + \sigma_j)),  R_i + R_j + R_{\mathrm{rounding},i} +
+            R_{\mathrm{rounding},j} + \frac{2.5}{2}
```

Comment:
What determines whether the factor is 2^(1/6) or 2.5, and where does the 2.5 come from? We may encourage more understanding by factoring out the parts of the equation that stay constant.

Reply:
`alpha` sets whether only the repulsive part of the potential is included, or the attractive part. Internally, this uses a cutoff of `2**(1/6)` for the repulsive only potentials (WCA) - cutoff at the minimum of the potential. Internally, it uses *no* cutoff for the attractive part. 2.5 \sigma is the traditional `r_cut` used for LJ simulations.

Reply:
It would be nice to reduce duplicated stuff like the sum of all the R's to highlight the term that is changing.

I experimented with putting this in a table, but even that is a little bit too large.

Reply:
I pushed a commit defining where the 2.5 and 2^1/6 come from

Reply:
There are a lot of common terms, but giving them names and reusing them only adds more layers of complexity for the user to understand.

Reply:
It's not clear to me whether users would find that more readable or not - e.g. if we gave names to R_i + R_j + R_rounding_i + R_rounding_j and (\sigma_i * \beta_i + \sigma_j * \beta_j).

As the equations are now, we would need to break them at some point, they run past the end of the rtd theme page width.

Reply:
The natural spot is to break after the comma in the max function